copier.yml

```diff
@@ -213,16 +234,6 @@ domains_prod:
       Leave empty to apply this rule on all available entrypoints. Those are defined
       in your Traefik deployment.
 
-    Example:
-
-    # Main domains, with TLS and cert resolved with "letsencrypt" resolver
-    - hosts: [www.main.com, shop.main.com]
-    # Alternate domains that redirect to main ones
-    - {hosts: [main.com, old.main.com], redirect_to: www.main.com}
-    - {hosts: [www.shop.main.com, old.shop.main.com], redirect_to: shop.main.com}
-    # VPN-only domain, with self-signed certificate
-    - {hosts: [vpn.main.com], cert_resolver: true}
-
     ⚠ Remember that you have to write one-line YAML, at least until Copier
     supports multiline input
```

Comment:
Wouldn't this comment become deprecated if we are using `multiline`?

Reply:
Indeed.

Reply:
Oh sorry, copier supports multiline only since the latest alpha, so this is still useful until a v6 is released.